internal/injector/config/config-go.go

```diff
@@ -0,0 +1,141 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package config
+
+import (
+	"errors"
+	"fmt"
+	"go/parser"
+	"go/token"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/DataDog/orchestrion/internal/injector/aspect"
+	"golang.org/x/tools/go/packages"
+)
+
+const OrchestrionToolGo = "orchestrion.tool.go"
```

Comment:
```suggestion
const OrchestrionToolGoFilename = "orchestrion.tool.go"
```
If it was only a local const I would not mind but you look like you replaced all off occurences with the variable and the name is not-so clear I think.